src/snowcli/cli/snowpark/common.py

```diff
@@ -47,34 +47,19 @@ class SnowparkObjectManager(SqlExecutionMixin):
     def _object_type(self):
         raise NotImplementedError()
 
-    def create(self, *args, **kwargs) -> SnowflakeCursor:
+    @property
+    def _object_execute(self):
         raise NotImplementedError()
 
-    def execute(self, expression: str) -> SnowflakeCursor:
+    def create(self, *args, **kwargs) -> SnowflakeCursor:
```

Comment:
In other places we use `deploy` command. Should we alige the naming?

Reply:
Yes